src/connectors/tagService.ts

```diff
@@ -363,6 +389,7 @@ export class TagService extends BaseService {
    * Determine if an user followed a tag or not by a given id.
    *
    */
+  // superceded / deprecated by isActionEnabled / setActionEnabled
```

Comment:
merge this instead of #2621 

some check functions here are superceded / deprecated will be removed some time after